build/updater.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
-cd query-builder
+cd /data/project/query-builder-test/query-builder
+git reset --hard origin/master
 git pull
-rm -rf node_modules
-npm install -g npm@6.14.8
-npm install
-npm run build
+./node_modules/.bin/npm install npm@6.14.8
```

Comment:
This cannot work the first time it runs, can it? There won't be an npm binary inside `node_modules/` so this should just use the system's npm for this step, right?!

Reply:
Done